react-native/android/app/src/main/java/io/keybase/ossifrage/modules/FileLogger.java

```diff
@@ -0,0 +1,89 @@
+package io.keybase.ossifrage.modules;
```

Comment:
This is akin to this: https://github.com/keybase/client/blob/master/desktop/app/console-helper.js

but for android.

It's not for the go side, it's to forward logs from JS to android and have android log it to a file